nix/util.nix

```diff
@@ -21,4 +21,15 @@ with lib; with haskell-nix.haskellLib;
   # cause errors when nix attempts to copy them into the store.
   removeSocketFilesFilter = _path: type:
     elem type ["regular" "directory" "symlink" ];
+
+  # Convert version strings from Cabal format (YYYY.M.D)
+  # to git tag format (vYYYY-MM-DD).
+  versionTag = cabal: let
+      parts = builtins.match "([[:digit:]]{4})\.([[:digit:]]{1,2})\.([[:digit:]]{1,2})" cabal;
+      leading0 = str: if stringLength str == 1 then "0" + str else str;
+    in
+      assert assertMsg (parts != null)
+        "versionTag: ${cabal} is not in YYYY.M.D format";
+      "v" + concatMapStringsSep "-" leading0 parts;
```

Comment:
This is simply a function declaration, not used anywhere yet, right? We have to be careful no breaking the Travis setup.

Reply:
Yes it's not used yet, but it could be used to make filenames more consistent, and remove the need to rename files in Travis.